nekoyume/Assets/_Scripts/Game/Game.cs

```diff
@@ -363,9 +391,10 @@ private IEnumerator CoInitializeTableSheets()
 
         private void ShowNext(bool succeed)
         {
-            IsInitialized = true;
+            Debug.Log($"[Game]ShowNext({succeed}) invoked");
             if (succeed)
             {
+                IsInitialized = true;
```

Comment:
is it intended?

Reply:
Yes it is. The `IsInitialized` field is only `true` if the `Agent` initialization was succeed. Otherwise, this game will be quit.